pkgs/top-level/all-packages.nix

```diff
@@ -39586,4 +39586,6 @@ with pkgs;
   udict = callPackage ../applications/misc/udict { };
 
   duden = callPackage ../applications/misc/duden { };
+
+  tests-stdenv-gcc-stageCompare = callPackage ../test/stdenv/gcc-stageCompare.nix { };
```

Comment:
please do not append entries to the end, they should be sorted to avoid merge conflicts.

Reply:
2929a9dea19b2de34f576601f8f14e1121986f28